lib/evm_database.rb

```diff
@@ -20,6 +20,9 @@ class EvmDatabase
   ORDERED_CLASSES = %w(
     RssFeed
     MiqWidget
+    MiqAction
+    MiqEventDefinition
+    MiqPolicySet
```

Comment:
@moolitayer Do you require any of these classes defined in this list?  They will be dynamically called from here: [lib/evm_database.rb#L47](https://github.com/ManageIQ/manageiq/blob/6f8773c8205d0d36d5ca914a51c92e439987f59d/lib/evm_database.rb#L47)